lib/stdlib/src/supervisor.erl

```diff
@@ -22,15 +22,28 @@
 -behaviour(gen_server).
 
 %% External exports
--export([start_link/2, start_link/3,
-	 start_child/2, restart_child/2,
-	 delete_child/2, terminate_child/2,
-	 which_children/1, count_children/1,
-	 check_childspecs/1, get_childspec/2]).
+-export([
+    start_link/2, start_link/3,
+    start_child/2,
+    restart_child/2,
+    delete_child/2,
+    terminate_child/2,
+    which_children/1,
+    count_children/1,
+    check_childspecs/1,
+    get_childspec/2
+]).
```

Comment:
I can see why you grouped start_link variants, but I am thinking you should not make this special case as they are technical different functions and it would be easier to just have one rule. I use to flatten my exports earlier but I find that having one per row is gives a better overview and avoids merge conflicts.

Reply:
Aha avoiding merge conflicts is good :)
I guess this would be true for anything we try to leave on the same line though.

Still it could be worth discussing flattening vs having each export on a separate line.
In this case it does seem as though we created at least a more flattened list.

Reply:
Well it is typically annoying for export lists as you most of the time want to keep both unrelated changes but it is quite easy to resolve the conflict incorrectly when the export list is a big not so structured blob of information. And as the younger people say all modern IDEs will let you collapse that part of the code anyway ;)

Reply:
The IDE might collapse it, but the merge conflict won't go away.
I hope that this way of formatting and grouping functions with the same name is a happy compromise.
But I do feel I am still missing you are trying to say?

Reply:
My point is that the merge conflicts will be rare if you have an export list with one function per
line but quite common if you have a few lines with many functions per line.  And I think the trade off between a long list
that I usually do not care to see and having more "compressed" list but get many merge conflicts, I prefer not having the merge conflicts. I was wondering if is worth the extra effort to group functions with the same name in the same line, as they are different functions even though they normally are related. But that is your trade off!

Reply:
Yes it is a trade off, I can hope that when functions have the same name, they are more likely to be edited at the same time, but that is pure speculation.